Replacing "Large" VM size with placeholder

Makopo opened the pull request:
Since some location (e.g. Japan East) doesn't available Large VM, replacing it with a placeholder would be better.
If you suggest any VM size suggestion instead, it should be available from every location.

azurecla commented:
Hi __@KylieLiang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Makopo committed:
Replacing "Large" VM size with placeholder

Since some location (e.g. Japan East) doesn't available Large VM, replacing it with a placeholder would be better.
If you suggest any VM size suggestion instead, it should be available from every location.
